library/Businessprocess/Web/Form/Validator/HostServiceTermValidator.php

```diff
@@ -60,10 +60,12 @@ public function isValid($terms)
 
             if ($this->parent->hasChild($term->getSearchValue())) {
                 $term->setMessage($this->translate('Already defined in this process'));
-            } else {
-                $testConfig->getNode('__unbound__')
-                    ->addChild($node);
+
+                return false;
```

Comment:
And what about the other terms? Shouldn't they be still validated as well? (They should)